sql/src/main/java/io/crate/execution/engine/pipeline/TableSettingsResolver.java

```diff
@@ -20,7 +20,7 @@
  * agreement.
  */
 
-package io.crate.operation;
+package io.crate.execution.engine.pipeline;
```

Comment:
I think pipeline is wrong here as this class contains helpers for working with the metadata, so maybe move it to the metadata package?

Reply:
It's only used in `ProjectionToProjectorVisitor` which is why I put it here (though `ProjectionToProjectorVisitor` could probably moved out of pipeline in a higher level package. But I think execution kinda makes sense?